draft-irtf-cfrg-opaque.md

```diff
@@ -777,8 +789,14 @@ envelope: The client's `Envelope` structure.
 
 ## Registration Functions {#registration-functions}
 
+This section contains definitions of the functions used by client and server
+during registration, including CreateRegistrationRequest, CreateRegistrationResponse,
+and FinalizeRegistrationRequest.
```

Comment:
Same here regarding backticks, though I see we are less consistent about function names and backticking them. Can you also backtick any occurrences of function names?